### PR TITLE
Update readme to include conda install

### LINKED ISSUE #52: [DOC] Add conda nightly builds to REAMDE's uild info
In build, let's add `conda install -c rapidsai-nightly cuspatial` so that interested parties don't only see that they have to build form source.

### PR DESCRIPTION
**Pull request by taureandyernv**
added conda install and light restructure to readme.  Closes #52

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,11 +21,15 @@ Future support is planned for the following operations.
 4. Grid-based indexing for points and polygons
 5. Quadtree-based indexing for large-scale point data
 6. R-Tree-based indexing for Polygons/Polylines
- 
-## Compile / Install C++ backend
-To compile and run cuSpatial, use the following steps.
 
-## Install dependencies
+## Install from Conda
+To insall via conda, please run:
```
Comment (harrism) [inline README.md]:
```suggestion
To install via conda:
```

File: README.md [hunk 2]
```diff
@@ -21,11 +21,15 @@ Future support is planned for the following operations.
...
+`conda install -c rapidsai-nightly cuspatial`
+
+## Install from Source
+To compile and install cuSpatial's C++ backend, please follow the following steps:
```
Comment (harrism) [inline README.md]:
```suggestion
To build and install cuSpatial from source:
```

File: README.md [hunk 3]
```diff
@@ -21,11 +21,15 @@ Future support is planned for the following operations.
...
+
+### Install dependencies
 
 Currently, building cuSpatial requires a source installation of cuDF. Install
 cuDF by following the [instructions](https://github.com/rapidsai/cudf/blob/branch-0.10/CONTRIBUTING.md#script-to-build-cudf-from-source)
```
Comment (harrism) [inline README.md]: This is out of date too, I think. @kkraus14 what is the procedure for building with the cuDF conda package now?

File: README.md [hunk 4]
```diff
@@ -34,7 +38,7 @@ The rest of steps assume the environment variable `CUDF_HOME` points to the
 root directory of your clone of the cuDF repo, and that the `cudf_dev` Anaconda
 environment created in step 3 is active.
 
-## Clone, build and install cuSpatial
+### Clone, build and install cuSpatial
 
 1. export `CUSPATIAL_HOME=$(pwd)/cuspatial`
```
Comment (harrism) [inline README.md]: These instructions are not complete either, since they don't cover building the Cython...

Comment (taureandyernv) [reply README.md]: true :)

File: README.md (hunk 3 above)
Comment (taureandyernv) [reply README.md]: I'll wait for the answer form @kkraus14 before resolving

**Comment by beckernick** [comment]
@taureandyernv what's the status of this PR? I agree with @harrism that we should fix the build instructions in a separate PR. We need to include the conda installation instructions in the README.

File: README.md (hunk 3 above)
Comment (taureandyernv) [reply README.md]: as @beckernick state here, https://github.com/rapidsai/cuspatial/pull/53#issuecomment-532369072, someone acutely familiar with the build can fix the README later.   I'll update the conda install portion to include other packages form rapidsai-nightly.  There are currently no conda install options in rapidsai

**Pull request merged**